functions/Add-DbaAgDatabase.ps1

```diff
@@ -211,6 +211,7 @@ function Add-DbaAgDatabase {
                 Write-Progress @progress
                 $testSplat = @{
                     SqlInstance            = $db.Parent
+                    SqlCredential          = $SqlCredential
```

Comment:
Thanks Andreas! $db.Parent is a connected object and shouldn't need SqlCredential. Does this really fix the issue?

Reply:
Perhaps a .Refresh() is needed somewhere instead?

Reply:
Thinking again, you are probably right. Could be the reason why I have not added that in the first round....

Reply:
Sometimes things are weird. If you want, you can use $db.Parent.Name and $SqlCredential to force a new connect or a refresh may help.